packages/jellyfish-api-core/src/category/oracle.ts

```diff
@@ -84,6 +84,20 @@ export class Oracle {
   async getOracleData (oracleid: string): Promise<OracleData> {
     return await this.client.call('getoracledata', [oracleid], 'number')
   }
+
+  /**
+   * Returns latest raw price updates from oracles.
+   *
+   * @param {OraclePriceFeed} priceFeed
+   * @return {Promise<OracleRawPrice[]>}
+   */
+  async listLatestRawPrices (priceFeed?: OraclePriceFeed): Promise<OracleRawPrice[]> {
+    if (priceFeed != null) {
+      return await this.client.call('listlatestrawprices', [priceFeed], 'number')
```

Comment:
Is there a better way to write this? 
If priceFeed value = empty object or null or undefined, this call will return an error.